Sources/Rendering/Core/PointPicker/index.d.ts

```diff
@@ -35,19 +37,21 @@ export interface vtkPointPicker extends vtkPicker {
 	 * 
 	 * @param {Vector3} p1 
 	 * @param {Vector3} p2 
-	 * @param {Number} tol 
+	 * @param {Number} tolerance 
+	 * @param {vtkProp3D} actor
 	 * @param {vtkMapper} mapper 
 	 */
-	intersectWithLine(p1: Vector3, p2: Vector3, tol: number, mapper: vtkMapper): number;
+	intersectWithLine(p1: Vector3, p2: Vector3, tolerance: number, actor: Nullable<vtkProp3D>, mapper: vtkMapper): number;
```

Comment:
in another class, `actor` is named `prop`. You might want to use the same naming